.travis.yml

```diff
@@ -18,6 +18,7 @@ env:
   - DB=mysqli
 
 before_install:
+  - sudo sh -c "echo 'precedence ::ffff:0:0/96 100' >> /etc/gai.conf"
```

Comment:
Other projects have been having the same issue. Anyone know how to contact
Travis' network team?
On Jul 15, 2015 7:00 PM, "Jordi Boggiano" <notifications@github.com> wrote:

> In .travis.yml
> <https://github.com/doctrine/dbal/pull/879#discussion_r34739742>:
>
> > @@ -18,6 +18,7 @@ env:
> >    - DB=mysqli
> >
> >  before_install:
> > +  - sudo sh -c "echo 'precedence ::ffff:0:0/96 100' >> /etc/gai.conf"
>
> What needs to happen is to make whichever network is losing v6 packets in
> the middle fix their shit. Not disabling things..
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/doctrine/dbal/pull/879/files#r34739742>.
>